### ISSUE TITLE
Allow POST for queries

### ISSUE THREAD

**Issue by alexisdimi** [paulovn/sparql-kernel#39]
A sparql query can easily get too large to fit in a GET request; `SPARQLWrapper` can be configured to do POST requests, but `sparql-kernel` does not seem to support them. Can this feature be added?

**Comment by paulovn** [comment]
Yes, that is easy. I've added it in the 1.3.0 branch, currently in testing, via a `%method` magic. As soon as it is tested, I will generate a new release.

**Comment by paulovn** [comment]
I've seen you have submitted a PR on this. Thanks!

**Comment by alexisdimi** [comment]
You got a PR for [this changeset](https://github.com/alexisdimi/sparql-kernel/commit/be7ecb63502a154875c071c7b90b8c2a8cd2be7c), that implements POST queries? Yeah I did it myself because I needed it, it was pretty easy after all. I didn't realize I had submitted the PR, though.

**Comment by alexisdimi** [comment]
I see the changeset got appended to the other, irrelevant PR... Clearly I don't know my way around this github feature yet! Anyway the patch has been working fine so far.

**Comment by paulovn** [comment]
Yes, it was included in the other PR, that's why I only noticed it later. Your patch is virtually identical to the solution in my branch, so it will work the same (there's not much space for imagination here, anyway :-)
